crates/runtime/src/capability/builtin.rs

```diff
@@ -211,23 +211,10 @@ impl std::fmt::Display for WrpcInterfaceTarget {
 #[derive(Clone, Debug, Eq, PartialEq)]
 /// Target entity
 pub enum TargetEntity {
-    /// Link target entity
-    Link(Option<String>),
-    /// Actor target entity
-    Actor(ActorIdentifier),
-    /// WRPC component
-    Wrpc(WrpcInterfaceTarget),
-}
-
-impl TargetEntity {
-    /// Tries to unwrap [`WrpcInterfaceTarget`]
-    pub fn try_unwrap_wrpc(self) -> Option<WrpcInterfaceTarget> {
-        if let Self::Wrpc(target) = self {
-            Some(target)
-        } else {
-            None
-        }
-    }
+    /// Component to invoke over the lattice using wRPC
+    Lattice(LatticeInterfaceTarget),
+    /// (unimplemented) Component to invoke locally
+    Local(LatticeInterfaceTarget),
```

Comment:
Is there any work planned regarding supporting "local" invocations and links? If not I'd say we should remove it and, in fact, remove enum altogether and use `LatticeInterfaceTarget` directly

Reply:
We do have work planned for the future ™️ but I will remove the unimplemented member and tag it as non_exhaustive so we can add it without breaking things in the future

Reply:
If we do indeed implement this feature, I would not expect users to have to explicitly select a "local" target - wasmCloud runtime should be able to perform the optimization transparently from the user's perspective.
In other words I would not expect to see any linking feature changes to enable "local" links - wasmCloud should be able to infer if it's running something on the prefix-instance-function tuple and if so, directly call that component without using NATS - this functionality would hook into `wrpc::Client` implementation and would not require any linking/targeting changes